src/concrete/ml/sklearn/base.py

```diff
@@ -242,8 +242,7 @@ def fhe_circuit(self) -> Optional[Circuit]:
         """Get the FHE circuit.
 
         The FHE circuit combines computational graph, mlir, client and server into a single object.
-        More information available in Concrete documentation:
-        https://docs.zama.ai/concrete/developer/terminology_and_structure#terminology
+        More information available in Concrete documentation (https://docs.zama.ai/concrete)
```

Comment:
Maybe https://docs.zama.ai/concrete/getting-started/terminology_and_structure would be better?

Reply:
ah yes sure